engines/librpma.c

```diff
@@ -59,94 +61,73 @@ enum librpma_io_mode {
 	FIO_RDMA_CHA_RECV
 };
 
-struct librpmaio_options {
+struct fio_librpmaio_options {
 	struct thread_data *td;
-	unsigned int port;
-	enum librpma_io_mode verb;
-	char *bindname;
+	unsigned int server_port;
```

Comment:
Can we use char* for port variable? As I see rpma (eg. rpma_conn_req_new) is using a port as char*.